src/authentification/components/inputs/InputField.tsx

```diff
@@ -0,0 +1,20 @@
+import { InputHTMLAttributes } from "react";
+import { useFormContext } from "react-hook-form";
+
+interface Props extends InputHTMLAttributes<HTMLInputElement> {
+  name: string;
+  placeholder: string;
+  validation: any;
+}
+
+export default function InputField({ name, placeholder, validation }: Props) {
+  const { register } = useFormContext();
+
+  return (
+    <input
+      {...register(name, { ...validation })}
+      className="h-[48px] sm:h-[32px] border input-padding outline-none rounded-[3px] border-input-border-gray"
+      placeholder={placeholder}
```

Comment:
You have extended the Props with InputHTMLAttributes, so you can do this 

```
export default function InputField({ name,validation, ...props }: Props) {
...
 <input
      {...register(name, { ...validation })}
      className="h-[48px] sm:h-[32px] border input-padding outline-none rounded-[3px] border-input-border-gray"
     {...props}
      />
```
Maybe you will need some other input prop so you dont need to add them separately.

Reply:
Put everything other than name and validation into {...props}.